Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/consent/SubmissionConsentViewModel.kt

```diff
@@ -30,8 +27,7 @@ class SubmissionConsentViewModel @AssistedInject constructor(
 
     fun onConsentButtonClick() {
         // TODO Do we have a Test registered at this time? We need to forward the decission with navargs?
-//        submissionRepository.giveConsentToSubmission(type = CoronaTest.Type.PCR)
-        analyticsKeySubmissionCollector.reportAdvancedConsentGiven()
+        // analyticsKeySubmissionCollector.reportAdvancedConsentGiven()
```

Comment:
How and when will analytics be added again? I think we can leave it there for now.

Reply:
It will be removed in case. advancedConsentGiven is now passed through the submission flow using navArgs since it is possible to have multiple pending tests (one antigene and one PCR at the same time)